packages/test-utils/src/generateNote.ts

```diff
@@ -0,0 +1,49 @@
+import { Note } from '@webb-tools/sdk-core';
```

Comment:
This note generation is very specific - I'm not convinced this should exist as a generic `generateMixerNote` test utility.

Reply:
Oh...Ok.
Should this PR be closed then?